.github/workflows/deploy-dev.yml

```diff
@@ -90,7 +90,7 @@ jobs:
             ## Branch preview
             ✅  Deploy successful!
 
-            https://${{ steps.extract_branch.outputs.branch }}--walletweb.review-wallet-web.5afe.dev
+            https://${{ steps.extract_branch.outputs.branch }}--walletweb.review-wallet-web.5afe.dev/home?safe=eth:0xA77DE01e157f9f57C7c4A326eeE9C4874D0598b6
```

Comment:
Added this so that we can open the app on an actual Safe instead of the Welcome page.